db/seeds.rb

```diff
@@ -5,3 +5,64 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+h1 = Hall.create(name: 'Hall 1', capacity: 20)
+h2 = Hall.create(name: 'Hall 2', capacity: 50)
+h3 = Hall.create(name: 'Hall 3', capacity: 50)
+h4 = Hall.create(name: 'Hall 4', capacity: 50)
+h5 = Hall.create(name: 'Hall 5', capacity: 50)
+h6 = Hall.create(name: 'Hall 6', capacity: 100)
+h7 = Hall.create(name: 'Hall 7', capacity: 100)
+h8 = Hall.create(name: 'Hall 8', capacity: 100)
+h9 = Hall.create(name: 'Hall 9', capacity: 100)
+h10 = Hall.create(name: 'Hall 10', capacity: 200)
+
+m1 = Movie.create(name: 'Harry Potter', description: 'Wizard movie', duration: 154)
```

Comment:
Please look into faker gem 
it will save a lot of time instead of writing out this data

Reply:
Thanks! I installed faker gem and created loops instead of single records :)